trulens_eval/examples/frameworks/llama_index/llama_index_quickstart.ipynb

```diff
@@ -89,10 +89,8 @@
    "source": [
     "from llama_index import VectorStoreIndex, SimpleWebPageReader\n",
     "\n",
-    "# llama_index 0.8.15 bug: need to provide metadata_fn\n",
```

Comment:
@joshreini1 can we also fix the version to a llama version that does not have this issue? I tried installing Friday and ran into this; but probably because I had a cached lower version